types/office-js-preview/index.d.ts

```diff
@@ -20384,7 +20384,8 @@ declare namespace Excel {
     interface BindingSelectionChangedEventArgs {
         /**
          *
-         * Gets the Binding object that represents the binding that raised the SelectionChanged event.
+         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. 
```

Comment:
The "Gets ..." terminology makes it sound like this is a description of a method or property, but there's no method or property signature below the description.

Reply:
@Rick-Kirkham There is a property there:

```
    interface BindingSelectionChangedEventArgs {
        /**
         *
         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. 
         * Use that ID with `BindingCollection.getItem(id)` to get the binding.
         *
         * [Api set: ExcelApi 1.2]
         */
        binding: Excel.Binding;

....

        /**
         *
         * Gets a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. 
         * Use that ID with `BindingCollection.getItem(id)` to get the binding.
         *
         * [Api set: ExcelApi 1.2]
         */
        binding: Excel.Binding;

```

Reply:
Strange. It doesn't appear for me in the diff for either of the events in both files. Here's a screenshot of the one just below this one:
![image](https://user-images.githubusercontent.com/9088056/85894314-587a7d80-b7a9-11ea-9022-81ab5a6bb636.png)

Reply:
I was having a similar issue earlier. Maybe it's a recent GitHub bug?